dhcpv6/option_userclass.go

```diff
@@ -23,29 +25,46 @@ func (op *OptUserClass) ToBytes() []byte {
 	buf := make([]byte, 6)
 	binary.BigEndian.PutUint16(buf[0:2], uint16(OPTION_USER_CLASS))
 	binary.BigEndian.PutUint16(buf[2:4], uint16(op.Length()))
-	// user-class-data has an internal data length field too..
-	binary.BigEndian.PutUint16(buf[4:6], uint16(len(op.UserClass)))
-	buf = append(buf, op.UserClass...)
+	for _, uc := range op.UserClasses {
+		binary.BigEndian.PutUint16(buf[4:6], uint16(len(uc)))
```

Comment:
I think this will always write to buf[4:6] which is not the desider behaviour.

Reply:
this is a copy-pasta error. Thanks for spotting :)